src/components/Header.jsx

```diff
@@ -0,0 +1,28 @@
+import { useState } from "react"
+
+function Header() {
+
+  const [loggeado, setLoggeado] = useState(false)
```

Comment:
Let's use a single language for consistency. Do not mix Spanish and English for variable names. Let's keep all of them in English

Reply:
I forgot to delete this component. I have replaced it for the NavMenu.